### DISCUSSION TITLE
How to setup Ruffle on xampp apache?

### DISCUSSION THREAD

**Discussion by josenoes** [ruffle-rs/ruffle#6105] [Q&A]
Ruffle doesn't work in xampp, I follow the steps but I can't get the school flash animations that I have to share them on the school intranet to run. I need help. Thanks

**Comment by danielhjacobs** [comment]
I suspect this is a MIME type issue, but rather than try to fix it, I suggest using the new official Ruffle CDN at https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle.

`<script src="https://cdn.jsdelivr.net/npm/@ruffle-rs/ruffle"></script>`

**Reply by josenoes** [reply]
Thanks for replying, but it still doesn't work.
